tests/reporter/kubernetes.go

```diff
@@ -244,6 +245,30 @@ func (r *KubernetesReporter) logVMIs(virtCli kubecli.KubevirtClient, vmis *v12.V
 	fmt.Fprintln(f, string(j))
 }
 
+func (r *KubernetesReporter) logDVs(virtCli kubecli.KubevirtClient) {
+
+	f, err := os.OpenFile(filepath.Join(r.artifactsDir, fmt.Sprintf("%d_dvs.log", r.failureCount)),
+		os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to open the file: %v\n", err)
+		return
+	}
+	defer f.Close()
+
+	dvs, err := virtCli.CdiClient().CdiV1beta1().DataVolumes(v1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to fetch pvcs: %v\n", err)
+		return
+	}
+
+	j, err := json.MarshalIndent(dvs, "", "    ")
+	if err != nil {
+		log.DefaultLogger().Reason(err).Errorf("Failed to marshal dvs")
+		return
```

Comment:
is there a reason why some of these errors are printed as a `fmt.Fprintf` and others use the logger pkg?

Reply:
No idea, but as I did a copy paste of logPVC, there are at least two more things I can do now. 
1. check this somewhere, and unify print
2. extract common `log` function (as it would be used at least 10 times) and just call it with dvs, pvcs, and so on.

Reply:
@davidvossel I do not want to do the things I have mentioned in previous message in current PR. Since I would like to have DV reporting merged.

But the two things mentioned here (check this somewhere, and unify print, extract common code) are worth investigating and doing. Do we have any issue/task for things like this?

Reply:
I have actually extracted the log function, so it is simpler now.